data/src/main/java/org/cryptomator/data/cloud/crypto/VaultConfig.kt

```diff
@@ -116,6 +116,15 @@ class VaultConfig private constructor(builder: VaultConfigBuilder) {
 			}
 		}
 
+		private fun getApiBaseUrl(hubClaim: Map<String, Any>): URI {
+			val apiBaseUrlKey = "apiBaseUrl"
+			return if (hubClaim.containsKey(apiBaseUrlKey)) {
+				parseUri(hubClaim, apiBaseUrlKey, ensureTrailingSlash = true)
+			} else {
+				parseUri(hubClaim, "devicesResourceUrl", ensureTrailingSlash = true).resolve("..")
```

Comment:
For future reference: `ensureTrailingSlash = true` is vital here.
If the `devicesResourceUrl` doesn't end with a slash, the returned `URI` will be `https://example.org/hub/` instead of `https://example.org/hub/api/`.